renderer/reducers/app.js

```diff
@@ -49,6 +52,14 @@ export function resetApp() {
   }
 }
 
+export const logout = () => async dispatch => {
```

Comment:
`await` is not used anywhere inside, nor does this function return a promise. consider removing `async` modifier

Reply:
Updated